docs/index.md

```diff
@@ -1,115 +1,217 @@
-            </summary>
-            <ul>
-                <li><a href="release_notes/ibexa_dxp_v4.5/#all-new-ibexa-commerce-packages">All-new Ibexa Commerce packages</a></li>
-                <li><a href="release_notes/ibexa_dxp_v4.5/#new-commerce-page-blocks">New commerce page blocks</a></li>
-                <li><a href="release_notes/ibexa_dxp_v4.5/#page-builder-for-b2b-portals">Page Builder for B2B portals</a></li>
-                <li><a href="release_notes/ibexa_dxp_v4.5/#personalization-improvements">Personalization improvements</a></li>
-                <li><a href="release_notes/ibexa_dxp_v4.5/#customer-data-platform-cdp-configuration">Customer Data Platform (CDP) configuration</a></li>
-                <li><a href="release_notes/ibexa_dxp_v4.5/#api-improvements">API improvements</a></li>
-            </ul>
-        </details>
-        <details>
-            <summary>
-                <img class="summary-icon" src="images/Reward.svg" height="25px" width="25px">Most popular pages</img>
-            </summary>
-            <ul>
-                <li><a href="api/php_api/php_api/">PHP API</a></li>
-                <li><a href="search/solr_search_engine/">Solr search engine</a></li>
-                <li><a href="search/search_api/">Search API</a></li>
-                <li><a href="content_management/content_model/">Content model</a></li>
-                <li><a href="content_management/images/images/">Images</a></li>
-                <li><a href="content_management/pages/page_blocks/">Page blocks</a></li>
-            </ul>
-        </details>
+    <div class="row">
+        <div class="col-12">
+            <div class="notification" id="tile2">
+                <div class="notification__content">
+                    <h2>The latest release is v4.5</h2>
+                    <div>The <a href="release_notes/ibexa_dxp_v4.5/">latest version of Ibexa DXP is v4.5</a>. You can now update your application to the latest version.</div>
+                </div>
+                <div class="notification__cta">
+                    <a href="update_and_migration/from_4.4/update_from_4.4/">Update your application</a>
+                </div>
+                <div class="notification__image">
+                    <svg><use xlink:href="images/icons.svg#notification-image" /></svg>
+                </div>
+            </div>
+        </div>
+        <div class="col-12">
+            <div class="accordion">
+                <details>
+                    <summary>
+                        <h2>Notable changes in v4.5</h2>
+                        <div class="accordion__toggler">
+                            <svg><use xlink:href="images/icons.svg#toggler" /></svg>
+                        </div>
+                    </summary>
+                    <div class="row">
+                        <div class="col-12 col-lg-6 col-fhd-3">
+                            <ul>
+                                <li><a href="api/php_api/php_api/">PHP API</a></li>
```

Comment:
The links have switched places: These should go under "Most popular pages", and links from most popular here.